src/test/java/ui/UiRendererTest.java

```diff
@@ -40,4 +65,109 @@ void printTaskForDay_noTasks_printsNoTasks() {
         assertEquals(outContent.toString(), EMPTY_TASK_DISPLAY_FORMAT);
     }
     
+    @Test
+    void printWeekHeader_forWeekView_printsWeekViewHeading() {
+        LocalDate today = LocalDate.now();
+        LocalDate startOfWeek = DateUtils.getStartOfWeek(today);
+        DateTimeFormatter dateFormatter = DateTimeFormatter.ofPattern("dd/MM/yyyy");
+        ByteArrayOutputStream outContent = new ByteArrayOutputStream();
+        System.setOut(new PrintStream(outContent));
+
+        printWeekHeader(startOfWeek, dateFormatter, false);
+
+        String expectedOutput = 
```

Comment:
Will the expected Output always be in this format? 
e.g. If the user does testing in 2025, will the expected output be the text given below?